imod/protocols/protocol_applyTransformationMatrix.py

```diff
@@ -110,15 +119,20 @@ def computeAlignmentStep(self, tsId):
             'output': self.getExtraOutFile(tsId),
             'xform': self.getExtraOutFile(tsId, ext=XF_EXT),
             'bin': binning,
-            'imagebinned': 1.0
+            'imagebinned': 1.0,
+            'taper': "1,0"
```

Comment:
I think imod is using 1,1 by default as you can see in other protocols

Reply:
Indeed. I added an option to choose whether to taper inside or outside, with the default being inside (1,1). What matters is that some tapering is done 🙂